client/packages/common/src/ui/layout/tables/material-react-table/useTableDisplayOptions.tsx

```diff
@@ -187,6 +213,10 @@ export const useTableDisplayOptions = <T extends MRT_RowData>(
           row.original?.['isSubRow'] && cell.column.id !== 'mrt-row-select'
             ? '2em'
             : undefined,
+        backgroundColor: column.getIsPinned()
+          ? // Remove transparency from pinned backgrounds
+            'rgba(252, 252, 252, 1)'
+          : undefined,
```

Comment:
The transparency comes from both the overall opacity and the background colour's alpha channel. The colour hard-coded here is the default colour minus the alpha value.